kshammer/spooti
# Initialize project with pyramid.

Shits not thread safe yo.

Haven't written any test yet too!!.

Anyways.... to keep the wheels moving....

## Changes
* injection of secrets and keys via environment variable can be done.
* /connect endpoint for searches.

## In order to test and make sure this works on your pc
* Activate your virtual environment
* Reinstall spooti package, made temp changes so that I could use the package in the web app. Someone should fix this up so that its thread safe.
* Starting from "Install the Project" in the README.txt in spooti_service. Follow the instructions there to initialize the project and install.
* To start the server (as mentioned from the README)
```
pserve development.ini
```
* Verify that you are able to to hit localhost:6543 w/o any errors
* Test the route /connect?start_artist=<ARTIST_NAME>&target_artist=<TARGET_ARTIST>
    * Defaults to Death Grips and KKB

## Beware
* When spamming refreshes, would get an Http Connection Error to spotify.....
